pom.xml

```diff
@@ -61,106 +66,73 @@
-                                    <inline>
-                                        <dependencySets>
-                                            <dependencySet>
-                                                <outputDirectory>/lib</outputDirectory>
-                                            </dependencySet>
-                                        </dependencySets>
-                                    </inline>
-                                </assembly>
-                                <args>
-                                    <!-- Labels used for image tagging -->
-                                    <BUILD_NUMBER>${project.version}</BUILD_NUMBER>
-                                    <BUILD_DATE>${maven.build.timestamp}</BUILD_DATE>
-                                    <GIT_REPO>${project.scm.url}</GIT_REPO>
-                                    <GIT_BRANCH>${git.branch}</GIT_BRANCH>
-                                    <GIT_COMMIT>${git.revision}</GIT_COMMIT>
-
-                                    <!-- Enable internet access -->
-                                    <http_proxy>${env.HTTP_PROXY}</http_proxy>
-                                    <https_proxy>${env.HTTPS_PROXY}</https_proxy>
-                                    <no_proxy>${env.NO_PROXY}</no_proxy>
-                                </args>
-                            </build>
-                        </image>
-                    </images>
-                    <verbose>true</verbose>
-                </configuration>
-            </plugin>
-        </plugins>
-    </build>
-
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>ch.qos.logback</groupId>
+                <artifactId>logback-access</artifactId>
+                <version>${logbackVersion}</version>
+            </dependency>
+            <dependency>
+                <groupId>ch.qos.logback</groupId>
+                <artifactId>logback-classic</artifactId>
+                <version>${logbackVersion}</version>
+            </dependency>
+            <dependency>
+                <groupId>ch.qos.logback</groupId>
+                <artifactId>logback-core</artifactId>
+                <version>${logbackVersion}</version>
+            </dependency>
+            <dependency>
+                <groupId>com.github.cafapi.logging</groupId>
+                <artifactId>caf-logging-common</artifactId>
+                <version>1.2.0-SNAPSHOT</version>
+            </dependency>
+            <dependency>
+                <groupId>com.github.cafapi.logging</groupId>
+                <artifactId>caf-logging-logback-access</artifactId>
+                <version>1.2.0-SNAPSHOT</version>
+            </dependency>
+            <dependency>
+                <groupId>com.github.cafapi.opensuse.tomcat</groupId>
+                <artifactId>caf-logging-tomcat-juli</artifactId>
+                <version>3.7.0-scmod-7821-new-SNAPSHOT</version>
```

Comment:
This should just be `3.7.0-SNAPSHOT`.

Reply:
While merging PR it will be updated to 3.7.0-SNAPSHOT

Reply:
This is a dependency in the same repository so it should be changed now.